backend/capellacollab/feedback/crud.py

```diff
@@ -13,13 +13,27 @@
 
 def count_feedback_by_rating(
     db: orm.Session,
-) -> dict[models.FeedbackRating, int]:
+) -> dict[tuple[models.FeedbackRating, bool], int]:
     return {
-        value[0]: value[1]
+        (value[0], value[1]): value[2]
         for value in db.execute(
             sa.select(
-                models.DatabaseFeedback.rating, sa.func.count()
-            ).group_by(models.DatabaseFeedback.rating)
+                models.DatabaseFeedback.rating,
+                sa.case(
+                    # pylint: disable=singleton-comparison
+                    (models.DatabaseFeedback.user_id != None, False),
+                    else_=True,
+                ).label("anonymous"),
+                # pylint: disable=not-callable
+                sa.func.count(),
+            ).group_by(
+                models.DatabaseFeedback.rating,
+                sa.case(
+                    # pylint: disable=singleton-comparison
+                    (models.DatabaseFeedback.user_id != None, False),
```

Comment:
```suggestion
                    (models.DatabaseFeedback.user_id.is_not(None), False),
```

This makes pylint happy and doesn't look worse.

Reply:
Ah, I tried out `is not None` earlier but that made mypy unhappy, I'll use this instead, thanks!